README.md

```diff
@@ -151,7 +151,7 @@ refer to [deployment instructions](https://datacube-explorer.readthedocs.io/en/l
 
 ### How do I modify the CSS/Javascript?
 
-The CSS is compiled from [Sass](https://sass-lang.com/), and the Javascript is compiled from
+The CSS is compiled from [Sass](https://sass-lang.com/), and the JavaScript is compiled from
```

Comment:
If you're fixing capitalization, there's a second "javascript" in the header that is also wrongly capitalized.

I think TypeScript in the next line is also mis-capitalized.